### DISCUSSION TITLE
How to get same image style as Doc D2 Tour?

### DISCUSSION THREAD

**Discussion by Dokiys** [terrastruct/d2#352] [Q&A] [answered]
While I according https://d2lang.com/tour/intro to start my first d2.
![image](https://user-images.githubusercontent.com/49900744/205543064-297968ae-a6d9-4a85-a3e2-bb29815086a3.png)
But I got different style.
![image](https://user-images.githubusercontent.com/49900744/205542873-39846e03-37e5-4c15-a676-202bb6849b32.png)

What should I do if I want to get the same result as in the Doc?

**Comment by alixander** [comment, marked as answer]
hi @Dokiys that render was generated with the TALA layout engine, which is paid. Most of the renders on the docs are with the free ones (dagre and elk), but the showcase ones, like that intro one and the one in the README, we put our best foot forward

**Reply by Dokiys** [reply]
Ok, thanks
